main/lsp/LSPTypechecker.cc

```diff
@@ -718,6 +731,10 @@ void LSPTypechecker::setSlowPathBlocked(bool blocked) {
     slowPathBlocked = blocked;
 }
 
+void LSPTypechecker::updateGsFromOptions(const DidChangeConfigurationParams &options) const {
+    this->gs->trackUntyped = options.settings->highlightUntyped.value_or(this->gs->trackUntyped);
```

Comment:
We may want to log some sort of warning when there are as-yet-unimplemented settings passed in the DidChangeConfigurationParams.

For example, `options.settings` might have things that `.has_value()` but will be ignored until someone wires those up, like `enableTypedFalseCompletionNudges`.

(Also, looking at the list of things in SorbetInitializationOptions, there are a handful of things in there that we may never support in this did change notification... I don't know whether it makes sense to make two separate types, but it may make sense to send some sort of warning back to the user if they attempt to set one of those things.)

Reply:
Hmm not sure what the API for this would be,  I don't think we send a response for this notification beyond flushing errors to the client? So unclear where this would go.

Reply:
you can either use window showMessage, or simply log something with the logger